tests/test_scylla.c

```diff
@@ -148,8 +151,51 @@ void test_scylla(void) {
   free_scylla_iota_transaction(&transaction);
   cass_cluster_free(cluster);
   cass_session_free(session);
+  scylla_api_logger_release();
 }
 
+void test_identity(void) {
```

Comment:
Here, we can see the importance of naming scheme. I can not identify which module the function belongs to.